connector/protobuf/src/test/scala/org/apache/spark/sql/protobuf/ProtobufFunctionsSuite.scala

```diff
@@ -22,7 +22,6 @@ import java.time.Duration
 import scala.jdk.CollectionConverters._
 
 import com.google.protobuf.{Any => AnyProto, BoolValue, ByteString, BytesValue, DoubleValue, DynamicMessage, FloatValue, Int32Value, Int64Value, StringValue, UInt32Value, UInt64Value}
-import org.json4s.StringInput
```

Comment:
Is this removed or deprecated?

Reply:
No, it's there. But method `JsonMethods#parse` signature changed. As this is the test code, I chose the simplest way to change it.